src/openhmd.c

```diff
@@ -13,6 +13,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
+#include <wchar.h>
```

Comment:
I think this include can be (re)moved?

Reply:
Yes.

Reply:
Windows ninja thing is [breaking](https://ci.appveyor.com/project/Wallbraker/openhmd/builds/33807989/job/6flt4dd9shp22jwu). IDK why. Works fine for me.

Reply:
So, `utils.h` is a header with function definitions. MSVC doesn't like the same function in multiple TUs in the same binary, so I'll make it static and see what the CI says.

Reply:
OK it's safe to look now. :)